README.md

```diff
@@ -6,10 +6,22 @@
 	<h4 align="center"> This is not the complete backend, but all main issues are resolved. Work left is to create some more endpoints for admin, securing of routes and level based access. </h4>
 	<h4 align="center"> index.html and index2.html are to test sockets, run them along with server and test their. Keep your dev console open. All things are checked but do testify everything that you think can crash and create issue. Much of left work is written in code but you can create issue for the same. </h4>
 
----
+
+[![forthebadge made-with-python](http://ForTheBadge.com/images/badges/made-with-python.svg)](https://www.python.org/) [![ForTheBadge built-with-swag](http://ForTheBadge.com/images/badges/built-with-swag.svg)](https://GitHub.com/D-E-F-E-A-T/) [![4U](https://forthebadge.com/images/badges/for-you.svg)](https://github.com/GDGVIT/)
+</br>
+[![GitHub stars](https://img.shields.io/github/stars/GDGVIT/AttendanceApp-Backend.svg?style=social&label=Star&maxAge=2592000)](https://GitHub.com/GDGVIT/AttendanceApp-Backend/stargazers/)[![GitHub followers](https://img.shields.io/github/followers/D-E-F-E-A-T.svg?style=social&label=Follow&maxAge=2592000)](https://github.com/D-E-F-E-A-T?tab=followers)
+[![GitHub repo size](https://img.shields.io/github/repo-size/GDGVIT/AttendanceApp-Backend.svg?logo=git&style=social)](https://github.com/GDGVIT/) [![GitHub code size in bytes](https://img.shields.io/github/languages/code-size/GDGVIT/AttendanceApp-Backend.svg?logo=python&style=social)](https://github.com/GDGVIT/AttendanceApp-Backend)
+ [![GitHub last commit](https://img.shields.io/github/last-commit/GDGVIT/AttendanceApp-Backend.svg?color=critical&logo=github&style=social)](https://github.com/GDGVIT/AttendanceApp-Backend/) [![GitHub contributors](https://img.shields.io/github/contributors/GDGVIT/AttendanceApp-Backend.svg)](https://GitHub.com/GDGVIT/AttendanceApp-Backend/graphs/contributors/) [![Open Source Love png3](https://badges.frapsoft.com/os/v3/open-source.png?v=103)](https://github.com/ellerbrock/open-source-badges/)
+ </br>
+ [![GitHub issues](https://img.shields.io/github/issues/GDGVIT/AttendanceApp-Backend.svg)](https://GitHub.com/GDGVIT/AttendanceApp-Backend/issues/) [![GitHub issues-closed](https://img.shields.io/github/issues-closed/GDGVIT/AttendanceApp-Backend.svg)](https://GitHub.com/GDGVIT/AttendanceApp-Backend/issues?q=is%3Aissue+is%3Aclosed)
+</br>
+[![GitHub pull-requests](https://img.shields.io/github/issues-pr/GDGVIT/AttendanceApp-Backend.svg)](https://GitHub.com/GDGVIT/AttendanceApp-Backend/pull/) [![GitHub pull-requests closed](https://img.shields.io/github/issues-pr-closed/GDGVIT/AttendanceApp-Backend.svg)](https://GitHub.com/GDGVIT/AttendanceApp-Backend/pull/)
+</br>
```

Comment:
Unnecessary badges added. Not required at this moment

Reply:
@Geek-ubaid 
They won't do any harm, It will be kind of you if they are not removed. Other repo pull request got closed for this.
If still has to be removed then tell, no additions will be made in future.

Reply:
These badges are included in the repo's which are well acclaimed. It won't do much harm but no point of including it in the readme also. However you can add it later if you want to.

Reply:
@Geek-ubaid Ok, will be removed.
:<